packages/data-point/src/Reducer.test.js

```diff
@@ -0,0 +1,58 @@
+jest.mock("./unique-id-scope", () => {
+  return () => () => "uniqueId";
+});
+
+const { Reducer, normalizeName } = require("./Reducer");
+const { IS_REDUCER } = require("./reducer-symbols");
+
+function createReducer() {
+  return new Reducer("type", "name", "spec");
+}
+
+describe("normalizeName", () => {
+  it("should append if provided", () => {
+    expect(normalizeName("name")).toEqual("name:uniqueId");
+  });
+
+  it("should only return unique id value if name is not provided", () => {
```

Comment:
I think this test name would be more accurate, because it only tests the positive case:

should ~~only~~ return unique id value if name is not provided